src/plugins/home/public/application/components/home_list_card.tsx

```diff
@@ -64,39 +64,50 @@ interface Config {
 }
 
 export const HomeListCard = ({ config }: { config: Config }) => {
+  const isMobile = useIsWithinBreakpoints(['xs', 's', 'm', 'l']);
+
+  const overflowStyle = {
+    overflow: isMobile ? 'scroll' : 'visible',
+    maxHeight: '230px',
+  };
+
   return (
     <>
-      <EuiPanel paddingSize="s" hasBorder={false} hasShadow={false}>
-        <EuiTitle>
-          <h4>{config.title}</h4>
-        </EuiTitle>
-        <EuiSpacer />
-        {config.list.length > 0 && (
-          <EuiDescriptionList>
-            {config.list.map((item) => (
-              <>
-                <EuiDescriptionListTitle>
-                  <EuiLink href={item.href} target="_blank">
-                    {item.label}
-                  </EuiLink>
-                </EuiDescriptionListTitle>
-                <EuiDescriptionListDescription>{item.description}</EuiDescriptionListDescription>
-              </>
-            ))}
-          </EuiDescriptionList>
-        )}
-
-        {config.allLink ? (
-          <>
-            <EuiSpacer />
-            <EuiLink href={config.allLink} target="_blank">
-              <EuiText size="s" style={{ display: 'inline' }}>
-                View all
-              </EuiText>
-            </EuiLink>
-          </>
-        ) : null}
-      </EuiPanel>
+      <EuiCard
+        textAlign="left"
+        title={
+          <EuiTitle>
+            <h4>{config.title}</h4>
+          </EuiTitle>
```

Comment:
Is it necessary to wrap with EuiTitle as it is already in the field of `title` of EuiCard.

Reply:
Thanks for the review. I tried discarding EuiTitle, but it resulted in less-than-ideal styling. Using EuiTitle can achieve the expected one with the Figma design .

Reply:
I c, thanks.